experimental/addons/istio/metadata.yaml

```diff
@@ -14,3 +14,5 @@ deployTo:
 
 dependencies:
 - name: fluxcd
+- name: rollout
```

Comment:
Why istio addon depends on rollout?? \cc @wangyikewxgm

Reply:
I think there is no need to depends rollout addon.

Reply:
@wangyikewxgm @wonderflow The istio addon's workflow `canary-rollout` depends on CRD `rollout` which is deployed by rollout addon.
![image](https://user-images.githubusercontent.com/105624645/208563752-148925ae-81c4-459f-91bf-60601e2c10de.png)
![image](https://user-images.githubusercontent.com/105624645/208563784-2b9c972a-d778-4676-ba29-368bce5fefe8.png)

Reply:
But user who use istio addon may not want to use rollout addon, depending on rollout addon is little bit confused. Can we separate this definition out as a single addon ?

Reply:
@wangyikewxgm This PR was proposed according to addon istio previous thinking. It is possible that rollout was not yet available as an addon at that time. If we want to separate this defintion out, a big change needs to be made to addon istio.I think we can first solve this problem in this way temporarily.

Reply:
There is no big change. I mean we can separate rollout related definition from istio addon to an another addon as `canary-rollout-istio` addon. And this addon depends on rollout and istio addon. What do you think of this solution？

Reply:
This solution will also work. It can solve the problem of strong dependency.